src/api/app/controllers/webui/apidocs_controller.rb

```diff
@@ -20,6 +20,6 @@ def file
       flash[:error] = "File not found: #{params[:filename]}"
       redirect_back(fallback_location: {controller: 'apidocs', action: 'index'})
     end
-    return
+    nil
```

Comment:
Please no. Having a return after the redirect is very common for controllers. At least it used to be.